src/components/form/_form.scss

```diff
@@ -96,3 +96,97 @@
     margin-bottom: $spacing-xs;
   }
 }
+
+@mixin form--experimental {
+  .#{$prefix}--fieldset {
+    @include reset;
+    margin-bottom: $spacing-xl;
+  }
+
+  .#{$prefix}--form-item {
+    @include font-family;
+    display: flex;
+    flex-direction: column;
+    flex: 1;
+    align-items: flex-start;
+  }
+
+  .#{$prefix}--form-item--light input,
+  .#{$prefix}--form-item--light input[type='number'] {
+    background: $field-02 !important;
```

Comment:
Any way to avoid the `!important`?

Reply:
this is just copied from the current codebase since I had to make a couple other tweaks. I can take a look, but not even sure what its doing.